packages/react-infinitegrid/test/TestHelper.js

```diff
@@ -1,5 +1,5 @@
 export function cleanHTML(html) {
```

Comment:
```suggestion
export function prettifyHTML(html) {
```

`cleanHTML`의 경우 HTML을 청소한다? 즉 삭제한다는 인상을 받았습니다.
`prettifyHTML`이 그 동작에 더 어울리지 않을까 싶습니다.